core/ledger/kvledger/pause_resume_test.go

```diff
@@ -133,10 +134,5 @@ func assertLedgerStatus(t *testing.T, provider *Provider, genesisBlocks []*commo
 }
 
 func contains(slice []int, val int) bool {
-	for _, item := range slice {
-		if item == val {
-			return true
-		}
-	}
-	return false
+	return slices.Contains(slice, val)
 }
```

Comment:
it is necessary not to change the body of the contains function, but the calls themselves. the contains function is no longer needed.